jobs/build/custom/Jenkinsfile

```diff
@@ -230,8 +230,9 @@ node {
 
             stage("build images") {
                 if (!any_images_to_build) { return }
+                buildlib.registry_quay_qe_login()
                 command = doozerOpts
-                command += "${include_exclude} --profile ${repo_type} images:build --push-to-defaults"
+                command += "${include_exclude} --profile ${repo_type} images:build --push-to-defaults --registry-config-dir=./qe_quay_config"
```

Comment:
This will be fine for v4, but for v3 we are still sending things to reg-aws, and this won't work for that. We still need v3 to work here, custom job builds the final release images with signed contents.

You could either fork the logic by version here in `custom`, or (I think might be cleaner) make a single `buildlib.qe_registry_login` routine that you invoke to login to whichever is appropriate to the version, and use that here and in ocp3/ocp4.